wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -0,0 +1,132 @@
+{% load wagtailadmin_tags i18n %}
+
+<ol class="w-editing-sessions__list">
+    {% for session in sessions|slice:":4" %}
+        <li>
+            {% fragment as avatar %}
+                {% avatar user=session.user classname="w-editing-sessions__avatar" %}
+            {% endfragment %}
+
+            {% if session.revision_id %}
+                {% fragment as avatar_decorated %}
+                    <div class="w-editing-sessions__decorated-avatar w-ping w-ping--critical">
+                        {{ avatar }}
+                        {% icon name="warning" %}
+                        <span class="w-editing-sessions__decorator-filler"></span>
+                    </div>
+                {% endfragment %}
+
+                {% if current_session.user == session.user %}
+                    {% blocktranslate trimmed asvar saved_new_version_message %}
+                        You saved a new version in another session
+                    {% endblocktranslate %}
+                {% else %}
+                    {% blocktranslate trimmed with user_name=session.user|user_display_name|default:_("System") asvar saved_new_version_message %}
```

Comment:
In existing code we seem to be using `user_display_name` with no fallback. I don’t know how to arrive at a situation where the fallback would be used. Deleting the user’s account perhaps?

If we need this kind of replacement consistently I wonder if we should create a new filter that comes with the fallback.

Reply:
In the editing sessions case, this can happen if the new revision is made programmatically without a `user` object attached, or if the user was deleted after the revision was created.